schema/url/visualizations.raml

```diff
@@ -18,3 +18,14 @@ types:
     type: VariableSpec
     properties:
       value: string
+
+  # think we should assume type is binWidth if not provided?
+  # for line plot do we want to only allow binWidth? if so, we'll need to split this..
```

Comment:
i think numBins makes sense for line plot. I have lots of points and i want 10. Seems like a reasonable desire.